deploy/docker/DOCKER-ENV.md

```diff
@@ -9,17 +9,20 @@ Edit all the fields according to your specific case.
   | DTAAS_DIR | '/Users/username/DTaaS' | Full path to the DTaaS directory. This is an absolute path with no trailing slash. |
   | SERVER_DNS | <http>_foo.com_</http> | The server DNS, if you are deploying with a dedicated server. Remember not use  <http:>http(s)</http:> at the beginning of the DNS string |
   | OAUTH_URL | <http>_gitlab.foo.com_<http/> | The URL of your Gitlab instance. It can be <http>_gitlab.com_<http/> if you are planning to use it for authorization. |
-  | CLIENT_ID | 'xx' | The ID of your server OAuth application |
-  | CLIENT_SECRET | 'xx' | The Secret of your server OAuth application |
+  | OAUTH_CLIENT_ID | 'xx' | The ID of your server OAuth application |
+  | OAUTH_CLIENT_SECRET | 'xx' | The Secret of your server OAuth application |
   | OAUTH_SECRET | 'random-secret-string' | Any private random string. This is a password you choose for local installation. |
   | username1 | 'user1' | The gitlab instance username of a user of DTaaS |
   | username2 | 'user2' | The gitlab instance username of a user of DTaaS |
-  | CLIENT_CONFIG | '/Users/username/DTaaS/deploy/config/client/env.js' | Full path to env.js file for client |
 
-:clipboard: Important points to note:
+<!-- markdownlint-disable MD046 -->
```

Comment:
The `tip` is not being rendered correctly in markdown viewer. Please revert to
`:clipboard: Important points to note:`

Reply:
Sure thing